Source/ROLib/Modules/ModuelROPayload.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace ROLib
+{
+    public class ModuleROPayload : ModuleDeployableSolarPanel
+    {
+        protected PartModule pmROTank;
+
+        public override void OnStart(StartState state)
+        {
+            base.OnStart(state);
+            pmROTank = part.Modules["ModuleROTank"];
+            if (state == StartState.Editor && pmROTank is ModuleROTank moduleROTank)
+            {
+                if (moduleROTank?.Fields["currentCore"] is BaseField cCbf)
+                {
+                    cCbf.uiControlEditor.onFieldChanged += OnCoreChanged;
+                }
+                moduleROTank.enableVScale = false;
+                moduleROTank.Fields[nameof(moduleROTank.currentVScale)].guiActiveEditor = false;
+            }
+        }
+
+        internal void OnCoreChanged(BaseField bf, object obj)
+        {
+            UpdateAnimationAndTracking(bf);
+            startFSM();
+        }
+        private void UpdateAnimationAndTracking(BaseField bf)
```

Comment:
Missing line breaks for this and all following methods.

Reply:
Fixed.  Will reopen PR to avoid an accidental merge commit.